### ISSUE TITLE
Integrate Open Graph for better Social sharing

### ISSUE THREAD

**Issue by raj454raj** [stopstalk/stopstalk-deployment#209]
https://developers.facebook.com/tools/debug/sharing/?q=https%3A%2F%2Fwww.stopstalk.com%2F

**Comment by morrme** [comment]
From my (limited) understanding of your project, this involves adding `meta` tags to `layout.html` . Would any other files be affected?

**Comment by raj454raj** [comment]
Hey @morrme , You are right ! We need to add meta tags in the layout files. The thing is StopStalk has multiple layout files (more profoundly known as templates) which are then loaded into the single layout.html file. So the aim for this issue is to come up with a generalized & clean way to embed meta tags into these templates. I haven't really tried solving this, but it would be great if you can try and send a PR for the same :)

**Comment by morrme** [comment]
@raj454raj If the tags were just added to layout.html,  wouldn't each template "inherit" the tags as they do with the other details there currently?

**Comment by raj454raj** [comment]
@morrme : Exactly, we don't want that to be inherited right ? We want separate descriptions and images for each template.
